roslibrust_codegen/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
```

Comment:
Since this will be published as a separate crate, needs LICENSE, README.md, extra fields under package